### ISSUE TITLE
Upgrade of cluster autoscaler from 1.23 to 1.24

### ISSUE THREAD

**Issue by mukundjalan** [kubernetes/autoscaler#5386]
I am trying to upgrade `cluster-autoscaler` from `1.23` to `1.24` (alongwith my AWS EKS cluster). I am unable to find helm-charts for application version `1.24` on [helm artifact hub](https://artifacthub.io/packages/helm/cluster-autoscaler/cluster-autoscaler). Can I reuse the same helm chart provided for `1.23` which version `9.21.1` or is the helm release for `1.24` missing yet?

Apart from this, what else do I need to update? From the docs [here](https://github.com/kubernetes/autoscaler/blob/cluster-autoscaler-1.24.0/cluster-autoscaler/cloudprovider/aws/README.md), I understand that I only need to update the IAM policy and image version, am I missing anything?

**Comment by djmcgreal-cc** [comment]
Any chance we can get an update on the status of the helm chart? Is it supposed to be tracking the autoscaler releases or is there a delay? Is an update planned?

Thanks!

**Comment by agconti** [comment]
@mukundjalan and @djmcgreal-cc  I solved this by specifying the 1.24 EKS image for the chart. 

You can find the latest release for EKS 1.24 in the releases section of this repo: https://github.com/kubernetes/autoscaler/releases?q=24&expanded=true. I pulled the image from that release and set it as image for the chart and things worked well on a new 1.24 cluster. My thinking is that they just haven't released a new chart version yet where that image is the default. 

image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.24.0

```terraform
resource "helm_release" "cluster_autoscaler" {
  depends_on = [null_resource.update_kubeconfig]
  name       = "cluster-autoscaler"
  namespace  = "kube-system"
  chart      = "cluster-autoscaler"
  version          = "9.13.1"
  repository       = "https://kubernetes.github.io/autoscaler"
  create_namespace = false

  set {
    type = "string"
    name = "image.tag"
    value = "v${local.cluster_version}.0"
  }
  # more omitted
}
```

**Comment by mukundjalan** [comment]
@agconti I indeed tried the same and (seemed like) it worked fine for me. I just wanted the contributors to confirm on the same so that I do not see any surprises in production.

**Comment by djmcgreal-cc** [comment]
Likewise. And additionally, I'd like to know if the helm chart is going to be updated. It's more ideal to just track the helm chart to ensure that the chart and image are being tested here in sync.

Thanks, it'd be good to confirm one way or the other so we can configure our dependency update scanning accordingly.

**Comment by djmcgreal-cc** [comment]
I notice the chart version was published as 9.24.0 from 9.21.something. Is it that you're matching the minor version of K8s to the minor version of the helm chart?
